tests/FSharpx.Tests/EnumTest.fs

```diff
@@ -23,4 +23,28 @@ let ``parse throws an exception when it fails to parse`` ()=
     let parseEnglish ()=
         let x = Enum.parse("English") : LanguageOptions
         ()
-    Assert.Throws<System.Exception>( TestDelegate( parseEnglish ) )  
\ No newline at end of file
+    Assert.Throws<System.Exception>( TestDelegate( parseEnglish ) )  
+
+[<Test>] 
+let ``isDefined returns true when valid simple enum value is checked`` ()=
+    Assert.IsTrue(Enum.isDefined LanguageOptions.CSharp && Enum.isDefined LanguageOptions.FSharp && Enum.isDefined LanguageOptions.VB)
```

Comment:
Minor: this could be divided into test cases.

Reply:
Test for every enum value? or, in the same unit test multiple Assert.IsTrue for every enum value?

Reply:
I meant something along the lines of:
```fsharp
[<Test>]
[<TestCase(LanguageOptions.CSharp, TestName="{m}(LanguageOptions.CSharp)")>]
[<TestCase(LanguageOptions.FSharp, TestName="{m}(LanguageOptions.FSharp)")>]
[<TestCase(LanguageOptions.VB,     TestName="{m}(LanguageOptions.VB)")>]
let ``isDefined returns true when valid simple enum value is checked`` (validEnumValue:LanguageOptions) =
    Assert.IsTrue(Enum.isDefined validEnumValue)
```
Test explore look with such approach:
![image](https://user-images.githubusercontent.com/8547855/81205396-ec5e7300-8fca-11ea-9e22-a8818ee2ed79.png)